.github/workflows/aikido-thread-check.yml

```
@@ -0,0 +1,282 @@
+name: aikido-thread-check
+
+on:
+  pull_request:
+    branches: [main]
+  pull_request_review:
+    types: [submitted, edited, dismissed]
+  pull_request_review_thread:
```

actionlint (1.7.12)

[error] unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)